cms/grading/tasktypes/Communication.py

```diff
@@ -31,6 +31,7 @@
 import logging
 import os
 import tempfile
+from functools import reduce
```

Comment:
I think there is a newline between imports and froms?

Reply:
I usually don't put any.